util/libcrypto.num

```diff
@@ -4819,7 +4819,7 @@ EVP_DigestSignUpdate                    ?	3_0_0	EXIST::FUNCTION:
 EVP_DigestVerifyInit_ex                 ?	3_0_0	EXIST::FUNCTION:
 EVP_DigestVerifyUpdate                  ?	3_0_0	EXIST::FUNCTION:
 BN_check_prime                          ?	3_0_0	EXIST::FUNCTION:
-EVP_PKEY_CTX_new_provided               ?	3_0_0	EXIST::FUNCTION:
+EVP_PKEY_CTX_new_provided               ?	3_0_0	NOEXIST::FUNCTION:
```

Comment:
Just remove it.

Reply:
Done - also had to rebase due to PR #10771